.prettierrc.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+// https://prettier.io/docs/en/options.html
+module.exports = {
+	arrowParens: 'always',
+	endOfLine: 'lf',
+	printWidth: 100,
+	singleQuote: true,
+	trailingComma: 'es5',
```

Comment:
We use `trailingComma: 'all'` in the main repo. Which do we want to standardise on and use here (and in [`@stylelint/config-prettier`](https://github.com/stylelint/prettier-config/pull/1/files#diff-d9a52777b34c9360910514f53aad911d))?

Reply:
Currently released eslint-config-stylelint uses ES6, so trailingComma of function argument cannot be used.
https://github.com/stylelint/eslint-config-stylelint/blob/11.1.0/eslintrc.js#L12

Specify `ecmaVersion: 2019` in individual `eslintConfig`?

Reply:
Let's release a new version of eslint-config-stylelint with `ecmaVersion: 2019`.

In the meantime, let's offload the decision about trailing commas to Prettier and use their default (which [as of version 2 is `es5`](https://prettier.io/blog/2020/03/21/2.0.0.html#change-default-value-for-trailingcomma-to-es5-6963httpsgithubcomprettierprettierpull6963-by-fiskerhttpsgithubcomfisker)):

> Change default value for trailingComma to es5
> There is a possibility that the default value will change to always (meaning even more trailing commas) in a future major version of Prettier as the JavaScript ecosystem further matures.

Please remove `trailingComma: 'es5'` from this pull request and I'll remove `trailingComma: 'all'` from `@stylelint/prettier-config`.

Reply:
I have removed trailingComma from .prettierrc.